report1.txt

```diff
@@ -1 +1,6 @@
 This is myt report 1.
+
+This is myt report 1.
+
+v
```

Comment:
Improve the performance of this code

Reply:
This logic having performance in consideration.